Assets/Scripts/Views/AssetViewer.cs

```diff
@@ -100,4 +118,55 @@ public void Lighten(GameObject prefab)
     {
         prefab.GetComponentInChildren<Image>().color = Color.white;
     }
+
+    public void MoveTo(GameObject prefab, Vector2 target, float speed, MovementTypes style)
+    {
+        moveCoroutine = StartCoroutine(MoveAsset(prefab, target, speed, style));
```

Comment:
This is more of an observation, but it seems with this implementation, only one `Asset` can be moving at a time. That is, if you wanted a `Cutscene` where the speaker and listener come into view at the same time, that would not be feasible. 

How does this change the state of this feature, if at all?

Reply:
The current iteration of the project doesn't have a need for two or more Assets to be moving simultaneously. That is, however, functionality to consider adding in the future. I could see instances in which it would be beneficial to have.